README.md

```diff
@@ -1,69 +1,105 @@
-# Project "Piper" Overview
+# Description
 
-An efficient software development process is vital for success in building business applications on SAP Cloud Platform or SAP on-premise platforms.
-SAP addresses this need for efficiency with project "Piper". The goal of project "Piper" is to substantially ease setting up continuous deployment processes for the most important SAP technologies by means of Jenkins pipelines.
+An efficient software development process is vital for success in building
+business applications on SAP Cloud Platform or SAP on-premise platforms.
+SAP addresses this need for efficiency with project "Piper". The goal of project
+"Piper" is to substantially ease setting up continuous deployment processes for
+the most important SAP technologies by means of Jenkins pipelines.
 
 Project "Piper" consists of two parts:
 
- * A [shared library][piper-library] containing steps and utilities that are required by Jenkins pipelines.
- * A set of [Jenkins pipelines][piper-pipelines] using the piper library to implement best practice processes.
+ * A [shared library][piper-library] containing steps and utilities that are
+   required by Jenkins pipelines.
+ * A set of [Jenkins pipelines][piper-pipelines] using the piper library to
+   implement best practice processes.
 
 ## What you get
 
-This repository contains Jenkins pipeline instructions in form of a `Jenkinsfile` (together with some auxiliary files) which require the [shared library][piper-library] of project "Piper" to run. They are available under `pipelines/<scenario>/Jenkinsfile` in this repository.
+This repository contains Jenkins pipeline instructions in form of a
+`Jenkinsfile` (together with some auxiliary files) which require the [shared
```

Comment:
Comma before 'which'
My suggestion: 
"`Jenkinsfile`, together with some auxiliary files, which require the..."

Reply:
I don't want to change this since other wise, the following 'which' might be misunderstood as referencing to the auxiliary files, and not to the Jenkinsfile.